math/FunMath.cpp

```diff
@@ -1 +1,23 @@
 #include "FunMath.h"
+
+using namespace std;
+
+int NextLeapYear(int current_year){
+	int curr = current_year;
+	while(true){
+		if((curr % 4 == 0 && curr % 100 != 0) || (curr % 4 == 0 && curr % 100 && curr % 400 == 0)){
+			return curr;
+		}
+		curr++;
+	}
+	return -1; //Should never get here
+}
+
+int Factorial(int n){
```

Comment:
Add function comments: what does this function do, what are the parameters/return values.